src/l10n/fr.json

```diff
@@ -0,0 +1,116 @@
+{
+    "translations": {
+        "Passwords"                                                                                 : "Mots de passe",
+        "HTTPS Required"                                                                            : "HTTPS Requis",
+        "reload this page"                                                                          : "Recharger cette page",
+        "documentation"                                                                             : "Documentation",
+        "This application requires %s in order to work safely."                                     : "Cette application dépend de %s afin de fonctionner correctement",
+        "Consult the %s to configure Nextcloud correctly."                                          : "Consultez %s afin de configurer Nextcloud correctement",
+        "If you do not have a HTTPS certificate, get one for free from %s."                         : "Si vous n’avez aucun certificat HTTPS vous pouvez en avoir un gratuitement un ici %s.",
+        "Enable %s to troubleshoot this error."                                                     : "Activez %s pour résoudre cette erreur.",
+        "You can try to %s the page with HTTPS."                                                    : "Vous pouvez essayer de %s la page avec HTTPS.",
```

Comment:
The `%s` here is the "reload this page" from line 5. I don't know if this has an influence on the spelling.

Reply:
![grafik](https://user-images.githubusercontent.com/8255857/47047372-ac8d2c00-d197-11e8-9a81-91d08a48b3d6.png)